### PR TITLE
added markdown in curation nodes

### LINKED ISSUE #111: Curation notebooks fail in pipeline
With the current pipeline setup, the curation notebooks [fail in automation](http://ml-pipeline-ui-kubeflow.apps.odh-cl1.apps.os-climate.org/pipeline/#/runs/details/453a84fa-6969-458b-beda-b77ac3670af6) in `preprocessing.pipeline`.

image

The reason could be that the extraction folder options configured `samples_4`, `samples_10` contain extracted pdfs for which annotations dont exist in the annotation file.

A workaround could be to create another folder on S3 which contains the files for which the annotation exists to run which can be configured in the `config.py` in the `DATA_S3_PREFIX` [here](https://github.com/os-climate/aicoe-osc-demo/blob/master/notebooks/demo2/config.py#L26) during the preprocessing.pipeline run

### PR DESCRIPTION
**Pull request by oindrillac**
added some instructions on which path to configure such that the pipeline run doesnt error out
resolves #111

### CODE REVIEW

File: notebooks/demo2/pdf_table_curation.ipynb [hunk 1]
```diff
@@ -46,14 +46,16 @@
     "\n",
     "For running the notebook in automation in an elyra pipeline, the environment variables can be updated in the notebook \"Properties\" in the pipeline UI or under `\"env_vars\"` in the `demo2.pipeline yaml` file.\n",
     "\n",
+    "For the purpose of running the elyra pipeline, you can configure the `DATA_S3_PREFIX` in the `config.py` as `\"corpdata/ESG/pipeline_run/samples_1\"` which consists of the pdf for which the annotation exists.\n",
```
Comment (chauhankaranraj) [inline notebooks/demo2/pdf_table_curation.ipynb:49]:
```suggestion
    "For the purpose of running the elyra pipeline, you can configure the `DATA_S3_PREFIX` in the `config.py` as `\"corpdata/ESG/pipeline_run/sample_1\"` which consists of the pdf for which the annotation exists.\n",
```

File: notebooks/demo2/pdf_text_curation.ipynb [hunk 2]
```diff
@@ -37,14 +37,16 @@
     "\n",
     "For running the notebook in automation in an elyra pipeline, the environment variables can be updated in the notebook \"Properties\" in the pipeline UI or under `\"env_vars\"` in the `demo2.pipeline yaml` file.\n",
     "\n",
+    "For the purpose of running the elyra pipeline, you can configure the `DATA_S3_PREFIX` in the `config.py` as `\"corpdata/ESG/pipeline_run/samples_1\"` which consists of the pdf for which the annotation exists.\n",
```
Comment (chauhankaranraj) [inline notebooks/demo2/pdf_text_curation.ipynb:40]:
```suggestion
    "For the purpose of running the elyra pipeline, you can configure the `DATA_S3_PREFIX` in the `config.py` as `\"corpdata/ESG/pipeline_run/sample_1\"` which consists of the pdf for which the annotation exists.\n",
```

**Review by chauhankaranraj — CHANGES_REQUESTED** [review]
Small typo in s3 prefix, but otherwise I'm able to run both extraction and curation steps with data prefix as `corpdata/ESG/pipeline_run/sample_1` :rocket:

**Comment by oindrillac** [comment]
Ah thanks for the catch @chauhankaranraj ! Fixed it 👍

**Review by chauhankaranraj — APPROVED** [review]
/lgtm /approve

**Pull request merged**